src/libs/ajax/leonardo/Disks.ts

```diff
@@ -61,6 +56,7 @@ export const Disks = (signal: AbortSignal) => {
   };
 };
 
+// TODO: these should be deletable?
```

Comment:
What do you mean by deletable here? Could you link the follow up JIRA ticket?

Reply:
Oh, these need to stay to assist with test typing. I'll rename though, the `ajax` part is what threw me off